google/cloud/spanner_dbapi/connection.py

```diff
@@ -141,14 +144,23 @@ def inside_transaction(self):
         """Flag: transaction is started.
 
         Returns:
-            bool: True if transaction begun, False otherwise.
+            bool: True if transaction started, False otherwise.
         """
         return (
             self._transaction
             and not self._transaction.committed
             and not self._transaction.rolled_back
         )
 
+    @property
+    def transaction_begun(self):
```

Comment:
nit: Here also, maybe better with `transaction_started`?

Reply:
Changed to client_transaction_started